src/components/SignatureSettings.vue

```diff
@@ -93,6 +104,27 @@ export default {
 			return identities
 		},
 	},
+	watch: {
+		async signatureAboveQuote(val, oldVal) {
+			await this.$store
+				.dispatch('patchAccount', {
+					account: this.account,
+					data: {
+						signatureAboveQuote: val,
+					},
+				})
+				.then(() => {
```

Comment:
watch out, you are mixing async/await with promises.

if you await the dispatch then wrap it with a try-catch for the error handling. you can then use the code as if it was synchronous

Reply:
omg, thx for pointing this out